tests/helper/test_helper.py

```diff
@@ -429,7 +431,10 @@ def test_notify_if_message_sent_outside_narrow(
 
     if footer_updated:
         report_success.assert_called_once_with(
-            "Message is sent outside of current narrow."
+            "Message is sent outside of current narrow. Press {} to narrow to message recipient. {}".format(
+                KEY_BINDINGS["NARROW_MESSAGE_RECIPIENT"]["keys"][0],
+                NARROW_TO_MESSAGE_RECIPIENT,
```

Comment:
As per discussion in the stream, let's skip the icon for now; I think it'd be a representation of the key somehow if we did, a little like we have `[P]`.

Reply:
Again, this is now #1203